components/event-publisher-proxy/cmd/event-publisher-proxy/main.go

```diff
@@ -1,21 +1,24 @@
 package main
 
 import (
+	"k8s.io/client-go/dynamic"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
 	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/application"
 	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/env"
-	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/handler"
+	"github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/handler/http"
```

Comment:
Here, http handler is basically the BEB handler. We also have [proxy-handler](https://github.com/kyma-project/kyma/pull/10523/files?file-filters%5B%5D=.go&file-filters%5B%5D=.mod&file-filters%5B%5D=.sum&hide-deleted-files=true#diff-b0e8438d638396e1f6e1719eee9dd43c37a3184409727b931116557917db828dR117) referring to this handler. Let's be consistent.